docs/quick_start.md

```diff
@@ -1,17 +1,157 @@
+| NPU   Name           | Health        | Power(W)    Temp(C)           Hugepages-Usage(page)|
+| Chip                 | Bus-Id        | AICore(%)   Memory-Usage(MB)  HBM-Usage(MB)        |
++======================+===============+====================================================+
+| 0     xxx            | OK            | 0.0         40                0    / 0             |
+| 0                    | 0000:C1:00.0  | 0           882  / 15169      0    / 32768         |
++======================+===============+====================================================+
+```
+
+
+## 2. Installation
+
+Prepare:
+
+```bash
+apt update
+apt install git curl vim -y
+# Config pypi mirror to speedup
+pip config set global.index-url https://mirrors.tuna.tsinghua.edu.cn/pypi/web/simple
+```
+
+Create your venv
+
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+You can install vLLM and vllm-ascend plugin by using:
+
+```bash
+# Install vLLM main branch (About 5 mins)
+git clone --depth 1 https://github.com/vllm-project/vllm.git
+cd vllm
+VLLM_TARGET_DEVICE=empty pip install .
+cd ..
+
+# Install vLLM Ascend Plugin:
+git clone --depth 1 https://github.com/vllm-project/vllm-ascend.git
+cd vllm-ascend
+pip install -e .
+cd ..
+```
+
+
+## 3. Usage
+
+After vLLM and vLLM Ascend plugin installation, you can start to
+try [vLLM QuickStart](https://docs.vllm.ai/en/latest/getting_started/quickstart.html).
+
+You have two ways to start vLLM on Ascend NPU:
+
+### Offline Batched Inference with vLLM
+
+With vLLM installed, you can start generating texts for list of input prompts (i.e. offline batch inferencing).
+
+```bash
+# Use Modelscope mirror to speed up download
+pip install modelscope
+export VLLM_USE_MODELSCOPE=true
+$ python3
+>>>
+```
+
+Try to execute below script to generate texts:
+
+```python
+from vllm import LLM, SamplingParams
+
+prompts = [
+    "Hello, my name is",
+    "The future of AI is",
+]
+sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+# The first run will take about 1.5 mins (10 MB/s) to download models
+llm = LLM(model="Qwen/Qwen2.5-0.5B-Instruct")
+
+outputs = llm.generate(prompts, sampling_params)
+
+for output in outputs:
+    prompt = output.prompt
+    generated_text = output.outputs[0].text
+    print(f"Prompt: {prompt!r}, Generated text: {generated_text!r}")
+```
+
+### OpenAI Completions API with vLLM
+
+vLLM can also be deployed as a server that implements the OpenAI API protocol. Run
+the following command to start the vLLM server with the
+[Qwen/Qwen2.5-0.5B-Instruct](https://huggingface.co/Qwen/Qwen2.5-0.5B-Instruct) model:
+
+```bash
+# Use Modelscope mirror to speed up download
+pip install modelscope
+export VLLM_USE_MODELSCOPE=true
+# Deploy vLLM server (The first run will take about 1.5 mins (10 MB/s) to download models)
+vllm serve Qwen/Qwen2.5-0.5B-Instruct
+```
+
+Once your server is started, you can query in a new terminal:
+```bash
+docker exec -it vllm-ascend-env bash
```

Comment:
is it OK to create another new bash in the same container? I'm not sure.

Reply:
we can start as backgroud process using `&`, and use `kill -2` stop vllm gracefully, but not sure it's to complex or not. see next commits.